R/database.R

```diff
@@ -1648,9 +1654,9 @@ get_coding_ssm = function(limit_cohort,
   }else{
     all_meta = get_gambl_metadata(from_flatfile = from_flatfile, seq_type_filter = seq_type)
   }
-  all_meta = dplyr::filter(all_meta,seq_type == {{ seq_type }})
 
-  all_meta = get_gambl_metadata(from_flatfile = from_flatfile, seq_type = seq_type)
+  all_meta = dplyr::filter(all_meta,seq_type == {{ seq_type }})
+  all_meta = get_gambl_metadata(from_flatfile = from_flatfile, seq_type_filter = seq_type)
```

Comment:
Should this line be removed? It seems like these two lines are attempting to accomplish the same thing different ways.

Reply:
Indeed, good catch!